Buildings/Fluid/ZoneEquipment/FanCoilUnit/Controls/ConstantFan_VariableWaterFlowrate.mo

```diff
@@ -0,0 +1,176 @@
+within Buildings.Fluid.ZoneEquipment.FanCoilUnit.Controls;
+block ConstantFan_VariableWaterFlowrate
+  "Controller for fan coil system with variable water flow rates and fixed speed fan"
+
+  parameter .Buildings.Controls.OBC.CDL.Types.SimpleController controllerTypeCoo=Buildings.Controls.OBC.CDL.Types.SimpleController.PI
+    "Type of controller" annotation (Dialog(group="Cooling mode control"));
+  parameter Real kCoo=1 "Gain of controller"
+    annotation(Dialog(group="Cooling mode control"));
+  parameter Real TiCoo=0.5 "Time constant of integrator block"
+    annotation(Dialog(group="Cooling mode control",
+      enable = controllerTypeCoo == Buildings.Controls.OBC.CDL.Types.SimpleController.PI or
+      controllerTypeCoo == Buildings.Controls.OBC.CDL.Types.SimpleController.PID));
+  parameter Real TdCoo=0.1 "Time constant of derivative block"
+    annotation(Dialog(group="Cooling mode control",
+      enable = controllerTypeCoo == Buildings.Controls.OBC.CDL.Types.SimpleController.PD or
+      controllerTypeCoo == Buildings.Controls.OBC.CDL.Types.SimpleController.PID));
+
+  parameter .Buildings.Controls.OBC.CDL.Types.SimpleController controllerTypeHea=Buildings.Controls.OBC.CDL.Types.SimpleController.PI
+    "Type of controller" annotation (Dialog(group="Heating mode control"));
+
+  parameter Real kHea=1 "Gain of controller"
+    annotation(Dialog(group="Heating mode control"));
+  parameter Real TiHea=0.5 "Time constant of integrator block"
+    annotation(Dialog(group="Heating mode control",
+      enable = controllerTypeHea == Buildings.Controls.OBC.CDL.Types.SimpleController.PI or
+      controllerTypeHea == Buildings.Controls.OBC.CDL.Types.SimpleController.PID));
+  parameter Real TdHea=0.1 "Time constant of derivative block"
+    annotation(Dialog(group="Heating mode control",
+      enable = controllerTypeHea == Buildings.Controls.OBC.CDL.Types.SimpleController.PD or
+      controllerTypeHea == Buildings.Controls.OBC.CDL.Types.SimpleController.PID));
+
+  parameter Real dTHys(
```

Comment:
why not using temperature difference?

Reply:
I have used the Real data type instead of the temperature difference data type, in compliance with the OBC conventions. I have a feeling Michael might ask us to move the controls to the OBC package, in which case the changes we would have to make will be minimal